### DISCUSSION TITLE
Reduce vertical space on home page

### DISCUSSION THREAD

**Discussion by danraymond** [nunocoracao/blowfish#557] [Ideas]
How can this space be reduced? (see image) please?

<img width="975" alt="reduce screen space" src="https://user-images.githubusercontent.com/1908612/221745489-5ecc890d-3eea-46ee-a630-f3874a81193f.png">

This has been solved  for my use case.

**Comment by nunocoracao** [comment]
You can override the page with a custom page for your use-case

**Comment by danraymond** [comment]
Hi Nuno,

Thank you, I have been learning to do that.

Regards

Dan
